client/package-lock.json

```diff
@@ -13187,6 +13187,14 @@
         }
       }
     },
+    "react-social-icons": {
```

Comment:
This might be a different situation, but I remember Saad always said not to commit changes to package-lock or package files, so this might be something to look into. I don't think you have to commit these changes, so you might be able to just roll back these files.